test/Driver/E.C

```diff
@@ -6,10 +6,9 @@
 // LICENSE.TXT for details.
 //------------------------------------------------------------------------------
 
-// RUN: %cling -dM -E -xc++ /dev/null | grep CLING | FileCheck %s
+// RUN: echo "" | %cling -dM -E -xc++ -std=c++11 | grep CLING | FileCheck %s
```

Comment:
I want to force c++11 for the __CLING__CXX11 check.
At some point (MSVC) c++14/17 may be the default?